jest.config.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
      */
     {
       ...sharedConfig,
-      modulePathIgnorePatterns: ["v2"],
+      displayName: "v1",
+      testPathIgnorePatterns: ["<rootDir>/src/v2"],
```

Comment:
I am not 100% sure about this but thinking what we need to exclude `/src/v2` from this jest _project_ may be `testPathIgnorePatterns` instead of `modulePathIgnorePatterns`. Want to double check with @damassi.

Reply:
Could we maybe update prev to `modulePathIgnorePatterns: ["<rootDir>/src/v2"]`? If not, and this works better, np! Was mostly fishing in the docs trying to set this up.

Reply:
I switched to use `modulePathIgnorePatterns: ["<rootDir>/src/v2"]` and got same result as the change here. From the doc it seems `testPathIgnorePatterns` is more aligned with what we want (i.e. skipping tests in v2), so I'll keep it for now. We can keep iterating. Thanks for reviewing!